app/views/foods/_food.html.erb

```diff
@@ -0,0 +1,8 @@
+<tr class="foods-tr">
+ <td><%= food.name %></td>
+ <td><%= food.measurement_unit %></td>
+ <td><%= food.price %></td>
+ <td> <%= button_to "Delete", food, method: :delete, class: "recipe-remove" %></td>
```

Comment:
- [ ] Again here as requested by the previous reviewer kindly ensure that the user who creates the food item is the only one who can delete that Item. Currently, I have two users and they can both delete food items even though the items were created by only one user

Reply:
Please note that we are team of 3 students, no 2. And we have specific requirements because of that. We can not verify who is an owner of "food", because there is no user_id in "Food" table based on ML diagram for 3 students.

Reply:
Also, I didn't find such requirement as "ensure that only the creator of a food can delete the food and not another user" in Business Requirements https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md
Could you please help me to find it?